tests/Kernel/fixtures/rendering.yml

```diff
@@ -905,6 +943,7 @@
       title: "Item title"
       url: "http://example.com"
       detail: "EU economy, the euro, and practical information for EU businesses and entrepreneurs."
+      lenght: 85
```

Comment:
Why are we adding all these `length` values to these tests? Existing tests shouldn't be changed, or else we risk changing behavior that we want to keep. In this case if length is not set then no trim should happen, do we still ensure that?

Reply:
They were not needed, i removed them